src/types/supabase.type.ts

```
@@ -461,6 +461,15 @@ export type Database = {
           is_liked: boolean;
         }[];
       };
+      get_user_like_plans: {
+        Args: { user_id_param: string };
+        Returns: Plan[];
+      };
+      get_all_plans: {
+        /* eslint-disable @typescript-eslint/no-empty-object-type */
+        Args: {};
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)